hash/fmtowns_cd.xml

```diff
@@ -1018,6 +1018,143 @@ User/save disks that can be created from the game itself are not included.
 		</part>
 	</software>
 	
+	<software name="eimmy">
+		<description>Eimmy to Yobanaide</description>
+		<year>1995</year>
+		<publisher>C's Ware</publisher>
+		<part name="cdrom" interface="fmt_cdrom">
+			<diskarea name="cdrom">
+				<disk name="eimmy to yobanaide" sha1="4c321b593322b94cf84a68ad1021d0cf3b24b1c4" />
+			</diskarea>
+		</part>
+	</software>
+	
+	<software name="elfish">
+		<description>Elfish</description>
+		<year>1994</year>
+		<publisher>Fujitsu</publisher>
+		<info name="usage" value="Requires HDD installation"/>
+		<part name="cdrom" interface="fmt_cdrom">
+			<diskarea name="cdrom">
+				<disk name="elfish" sha1="c0dd2d0c0caaea9098de8a550aac7b5005030f8f" />
+			</diskarea>
+		</part>
+	</software>
+	
+	<software name="elfishlt">
+		<description>Elfish Lite</description>
+		<year>1993</year>
+		<publisher>Fujitsu</publisher>
+		<part name="cdrom" interface="fmt_cdrom">
+			<diskarea name="cdrom">
+				<disk name="elfish" sha1="1667eb91450e756ae37910aa9291957430fc588b" />
```

Comment:
Both elfish and elfishlt want to have a disk called "elfish.chd" in the file system, but with different checksums. That won't work, please rename one of them.